libs/fetcher.js

```diff
@@ -335,7 +335,14 @@ Fetcher.middleware = function (options) {
                     }
                     if (err) {
                         var errResponse = getErrorResponse(err);
-                        res.status(errResponse.statusCode).json(responseFormatter(req, res, errResponse.output));
+                        if (req.query && req.query.returnMeta) {
```

Comment:
i think we are fine.  We are already doing req.query.returnMeta logic branching on line 348 below